environment.yml

```diff
@@ -0,0 +1,35 @@
+name: mne
+channels:
+- defaults
+dependencies:
+- python
+- pip
+- mkl
+- numpy
+- scipy
+- matplotlib
+- scikit-learn
+- mayavi
+- pandas
+- h5py
+- statsmodels
+- nose
+- pytest
+- pytest-cov
+- joblib
+- jupyter
+- sphinx
+- spyder
```

Comment:
I agree that spyder is probably a good idea for users but maybe an overkill for testing?

Reply:
I don't think it adds much download time. And in the meantime this sets us up to properly test the purpose of this (set up a usable environment for users that gets tested by CIs)